client/extensions/woocommerce/app/dashboard/store-location-setup-view.js

```diff
@@ -134,6 +129,15 @@ class StoreLocationSetupView extends Component {
 			// No need to set isSaving to false here - we're navigating away from here
 			// and setting isSaving to false will just light the button up again right
 			// before the next step's dialog displays
+
+			// If we don't support a calypso experience yet for this country, let
+			// them complete setup with the wp-admin WooCommerce wizard
+			if ( ! isStoreManagementSupportedInCalypsoForCountry( this.state.address.country ) ) {
+				const storeSetupURL =
```

Comment:
This might be a good thing to add some `tracks` around right here. Or an mc stat bump at least.

Reply:
Yeah, agreed!